File: mbrs/src/main/java/com/ftn/mbrs/service/PunjenjeService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::Punjenje


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Punjenje;

public interface PunjenjeService {

	public Punjenje save(Punjenje punjenje, Long voziloId, Long tipPrikljuckaId, Long stanicaId);
	
	public Punjenje update(Punjenje punjenje);

	public String delete(Long id);
	
	public Punjenje findOne(Long id);
	
	public List<Punjenje> findAll();
}

File: mbrs/src/main/resources/static/js/services/PunjenjeService.js
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::Punjenje

angular.module('gremoNaElektrikoApp.PunjenjeService', [])
	.factory('PunjenjeService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/punjenje');
		}
		
		factory.createPunjenje = function(punjenje , voziloId, tipPrikljuckaId, stanicaId) {
			return $http.post('/gremoNaElektriko/punjenje'+'/'+voziloId+'/'+tipPrikljuckaId+'/'+stanicaId, punjenje);
		}

		factory.deletePunjenje = function(id) {
			return $http.delete('/gremoNaElektriko/punjenje/'+id);
		}
		
		factory.updatePunjenje = function(punjenje) {
			return $http.put('/gremoNaElektriko/punjenje', punjenje);
		}
		
		return factory;
	});